config/rbac/role.yaml

```diff
@@ -5,18 +5,6 @@ metadata:
   creationTimestamp: null
   name: odh-model-controller-role
 rules:
-- apiGroups:
```

Comment:
Some changes here look like a rebase issue, since were recently added.

Reply:
Looks like this was added manually, rather than using kubebuilder comments. Thus, `make manifests` would remove this.

I'll create a PR fixing this.

Reply:
https://github.com/opendatahub-io/odh-model-controller/pull/158